platform/ios/PolyPodApp/PolyPod/ContentView.swift

```diff
@@ -160,8 +161,8 @@ struct ContentView: View {
 
     private func featureState(_ feature: Feature) -> ViewState {
         ViewState(
-            backgroundColor: feature.primaryColor,
-            borderColor: feature.borderColor,
+            backgroundColor: Color(fromHex: feature.primaryColor),
```

Comment:
Are we checking somewhere that it's hex only?

Reply:
No, we are not performing any checks. It is assumed that FeatureManifest contains colours in correct format.

Reply:
I seem to remember I opened a Jira ticket to create manifest static checks... Maybe we could revive that.